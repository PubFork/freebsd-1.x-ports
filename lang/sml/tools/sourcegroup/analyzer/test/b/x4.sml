signature B = sig structure A:A val b:A.t end

and A = sig type t end
