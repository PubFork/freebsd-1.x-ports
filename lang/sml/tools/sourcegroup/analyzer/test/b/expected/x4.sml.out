source sml "b/x4.sml"
  import signature A 
  export signature A B ;

